### DISCUSSION TITLE
Why these processes still remain in my system?

### DISCUSSION THREAD

**Discussion by noeltong** [jax-ml/jax#15237] [Q&A]
Here is the processes...

I'm sure that we have finished the running but these processes still remain in my system. Did i miss something in my code to kill these processes?

![image](https://user-images.githubusercontent.com/46706611/227967318-6192a45e-133d-4e96-9bf4-68b60782c839.png)

I'm now using `jax  0.3.25` and `jaxlib 0.3.25+cuda11.cudnn82`.

**Comment by jakevdp** [comment]
Thanks for the question! These processes look like they're coming from [joblib](https://pypi.org/project/joblib/) and [loky](https://pypi.org/project/loky/), neither of which are used by JAX or any of its dependencies, as far as I know. You might look at what other code you're executing that may be calling APIs from these libraries.

**Reply by noeltong** [reply]
Thank you for your kindly reply! I will further look the part of my code using joblib and loky.
